components/alert/alert.css

```diff
@@ -0,0 +1,47 @@
+@import "/default.css";
+
+
+body {
+    margin: 2rem;
+    font-family: 'Lato', 'Helvetica Neue', Helvetica, sans-serif;
+}
+
+.icon {
+    font-size: 1.2rem;
+    padding-right: 1rem;
+}
+
+.wrapper {
+    display: flex;
+    flex-direction: column;
+}
+
+
+.alert {
+    max-width: 600px;
+    padding: 1rem;
+    margin-bottom: 1rem;
+    font-size: 1rem;
+    border-radius: 5px;
+    align-items: center;
+}
+
+.alert-error {
+    color: var(--danger-color);
```

Comment:
check the contrast ratio of colors, it's not clearly visible in red alert. everything else is good!